src/reducers/profileReducer.ts

```diff
@@ -0,0 +1,36 @@
+import { IAction } from "../actions";
+import { ProfileActions } from "../actions/profileActions";
+
+export interface IProfile {
```

Comment:
Shall we move IProfile, UserMode and defaultsState to the state in "src/state/index.ts" or to a state file dedicated to Profile?

Reply:
I remaned IProfile to ProfileState and moved interfaces and enums to ProfileType.d.ts
I kept the defaultState in the reducer because only reducer is using it and its an object and not a type so cant place in d.ts file

Reply:
I renamed IProfile to ProfileState and moved ProfileState and UserMode to ProfileTypes.tsx file but left the default state in the reducer bcz only reducer is using it and it is an object and not a type